arches_lingo/src/arches_lingo/components/scheme/report/SchemeLabel.vue

```diff
@@ -137,75 +137,79 @@ async function update(tileId: string | undefined) {
 </script>
 
 <template>
```

Comment:
Is this file going away? (Shall we remove now?)

Reply:
yeah I wanted to get the removals in once everything refactored. Still this PR just a later iteration 👍

Reply:
thanks, just wanted to know if I needed to look at the diff :-)